sopel/modules/url.py

```diff
@@ -27,8 +27,16 @@
 except ImportError:
     from urlparse import urlparse
 
-USER_AGENT = 'Sopel/{} (https://sopel.chat)'.format(__version__)
-default_headers = {'User-Agent': USER_AGENT}
+USER_AGENT = (
+    'Mozilla/5.0 (Windows NT 10.0; Win64; x64) '
+    'AppleWebKit/537.36 (KHTML, like Gecko) '
+    'Chrome/78.0.3904.108 Safari/537.36'
+)
+default_headers = {
```

Comment:
Since you're touching this anyway, I wouldn't say no to fixing the name so it looks properly constant-like (all-caps).

Reply:
I'm not a huge fan of "constant"-like in Python anyway, and it would require changing other references further on in the plugin. But I'm not opposed to it entirely, I just figured this would be *really* fixed in the future.

Reply:
Per the PEP8 (and pylint) style, any module-level attribute should be all-caps.

Reply:
Right. The fact that it's in basic `snake_case` is an historical mistake. Not as bad as others in the Sopel codebase, but still a stylistic mistake.